src/usbipd.rs

```diff
@@ -66,9 +66,6 @@ pub struct UsbDevice {
 
     #[serde(rename = "PersistedGuid")]
     pub persisted_guid: Option<String>,
-
-    #[serde(rename = "StubInstanceGuid")]
-    pub stub_instance_id: Option<String>,
```

Comment:
Removing this is incorrect, since `serde` derives deserialization code from this struct. Annotate the struct with `#[allow(dead_code)]` instead.